ckan/config/solr/schema.xml

```diff
@@ -96,6 +96,13 @@ attribute with the form `ckan-X.Y` -->
       </analyzer>
     </fieldType>
 
+    <fieldType 
+        name="location_rpt" class="solr.SpatialRecursivePrefixTreeFieldType"
```

Comment:
I'm curious why you chose this instead of the `RptWithGeometrySpatialField` model.

Reply:
Is the oe from ckanext-spatial [docs](https://github.com/ckan/ckanext-spatial/blob/master/doc/spatial-search.rst#choosing-a-backend-for-the-spatial-search) (_solr-spatial-field_) recommends @EricSoroos.

Reply:
@EricSoroos we are early in our investigation of the different Solr fields and options so we just went along with the first option suggested in the [Solr docs](https://solr.apache.org/guide/8_11/spatial-search.html#rpt). Do you have experience working with these fields? Seems like `RptWithGeometrySpatialField` provides greater accuracy at the expense of a bigger index size (because it stores geometries)?